cyhy/core/geoloc.py

```diff
@@ -9,7 +9,37 @@
 GEODB_FILES = ["GeoIP2-City.mmdb", "GeoLite2-City.mmdb"]
 GEODB_CITY_PATHS = ["/usr/share/GeoIP/", "/usr/local/share/GeoIP/"]
 GEODB_FILE_PATHS = []
-RESTRICTED_COUNTRIES = ["China", "Iran", "North Korea", "Russia"]
+
+# The MaxMind GeoLite2 and GeoIP2 databases that we support use 
+# https://www.geonames.org/countries/ as the data source for country names.
+# For more info, see: 
+# https://support.maxmind.com/hc/en-us/articles/4414877149467-IP-Geolocation-Data#h_01FRRNFD5Z5EWNCAXM6SZZ5H2C
```

Comment:
If we cannot provide a link or document that sources the countries listed here I would at least like some kind of attribution for who provided them (team/name/whatever).

Reply:
I addressed this concern with additional information. here is the commit:
https://github.com/cisagov/cyhy-core/pull/91/commits/98125a8f98d2ad91daf1d9bcfb2f2297ca40dcbe

Reply:
Great, thanks!